news/3979.bug

```diff
@@ -0,0 +1,4 @@
+Do not bound the database session to the TransactionalSessionMaker class.
```

Comment:
Should we specify which database session?

Reply:
This is the session created in the TransactionalSessionMaker class, no sure how I can be more specific. Idea ?

Reply:
What about:
````
Do not bound the database session created using TransactionalSessionMaker class to the object created
````